specifyweb/frontend/js_src/lib/components/DataModel/saveBlockers.tsx

```diff
@@ -69,7 +67,7 @@ export const propagateBlockerEvents = (
 export function useSaveBlockers(
```

Comment:
the conflict with save blockers that resulted in the need for blockerKey, was that a conflict between multiple places setting blockers, or the same place overwriting the blockers it set before?

if it was a conflict between multiple different places, then in the useSaveBlockers, you could have just used `const id = useId('blocker'); const blockerKey = id(field?.name ?? '')` as your blocker key